pkg/cmd/controller/image.go

```diff
@@ -127,6 +127,9 @@ func (u podSpecUpdater) Update(obj runtime.Object) error {
 	case *kbatchv1.Job:
 		_, err := u.kclient.BatchV1().Jobs(t.Namespace).Update(context.TODO(), t, kmetav1.UpdateOptions{})
 		return err
+	case *kbatchv1.CronJob:
+		_, err := u.kclient.BatchV1().CronJobs(t.Namespace).Update(context.TODO(), t, kmetav1.UpdateOptions{})
+		return err
```

Comment:
@dmage @ricardomaraschini @openshift/openshift-team-build-api 

CronJob is GA in v1.21

Reply:
/cc @soltysh